v2/apiserver/internal/lib/restmachinery/request_context_filter.go

```diff
@@ -0,0 +1,43 @@
+package restmachinery
+
+import (
+	"context"
+	"net/http"
+)
+
+// requestContextFilter is a filter that can decorate an HTTP handler function
+// such that requests are examined and IFF the request utilizes an HTTP method
```

Comment:
```suggestion
// such that requests are examined and IF the request utilizes an HTTP method
```

Reply:
That was deliberate... IFF is a common shorthand for "if and only if," but perhaps it is not as well-known as I thought... in which case, I should probably change it.

Reply:
Wording is revised in f6fdbf303312266f646f712203d1ef2de335f8ec.

Reply:
Oh!  TIL.  hehe at least now I'm savvy :)